examples/client/Locomotion/src/Components/Marker/styled.js

```diff
@@ -58,6 +59,8 @@ export const MarkerContainer = styled.View`
 `;
 
 export const InfoBox = styled.View`
+  margin-top: ${Platform.OS === 'ios' ? '-45px' : '100px'};
```

Comment:
Can we find better solution for it? I don't want to do styling based on platform

Reply:
i cant figure out a better way